src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from "@angular/core";
+
+@Component({
+    standalone: true, 
```

Comment:
you can use ng lint to check indentation for you

Reply:
Ok, I'll learn about it